src/test/java/core/basesyntax/strategy/PurchaseHandlerTest.java

```diff
@@ -0,0 +1,40 @@
+package core.basesyntax.strategy;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.OperationHandlerService;
+import java.util.Map;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class PurchaseHandlerTest {
+    private OperationHandlerService operationHandlerService;
+    
+    @BeforeEach
+    void setUp() {
+        operationHandlerService = new PurchaseHandler();
+        Storage.fruitsStorage.clear();
+    }
+
+    @Test
+    void purchaseOperationNotEnoughAmount_NotOk() {

```

Comment:
bad test naming see [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#remember-about-naming-of-test-methods)